.github/contributing.md

```diff
@@ -109,6 +109,36 @@ Unit tests are located inside `__tests__`. Consult the [Jest docs](https://jestj
 - Write a unit test whenever possible
 - If a test is specific to a browser, create an e2e (end to end) test and make sure to indicate it on the test
 
+## Contributing Docs
+
+Currently, all the docs can be found in `packages/docs`. It has a English version by default, and translation(s) in corresponding `<lang>` sub-folder(s):
+
+- `zh`: Chinese translation.
```

Comment:
I think we should add a link to the translation folder here as other translations might be hosted in a different repository. This will make more sense after adding an external translation